File: pset1/mario2/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Ask the user for a height between 1-8
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // Print the piramids
    for (int i = 0; i < n; i++)
    {
        // Print the left spaces
        for (int j = 0; j < n - 1 - i; j++)
        {
            printf(" ");
        }
        // Print the left hashes
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // Print the gap
        for (int j = 0; j < 2; j++)
        {
            printf(" ");
        }
        // Print right hashes
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // Change of line
        printf("\n");
    }
}

File: pset6/mario/more/mario.py
# Ask the user for a height between 1-8
while True:
    n = input("Height: ")
    try:
        n = int(n)
        if n >= 1 and n <= 8:
            break
    except:
        continue


# Print the piramids
for i in range(n):

    # Print the left spaces
    for j in range(n - 1 - i):
        print(" ", end="")

    # Print the left hashes
    for j in range(i + 1):
        print("#", end="")

    # Print the gap
    for j in range(2):
        print(" ", end="")

    # Print right hashes
    for j in range(i + 1):
        print("#", end="")

    # Change of line
    print()